ipatests/test_integration/test_commands.py

```diff
@@ -699,3 +699,61 @@ def is_tls_version_enabled(tls_version):
         assert not is_tls_version_enabled('tls1')
         assert not is_tls_version_enabled('tls1_1')
         assert is_tls_version_enabled('tls1_2')
+
+    def test_sss_ssh_authorizedkeys(self):
+        """
+        When there is ssh key is added for IPA user then
+        'sss_ssh_authorizedkeys ipauser' should return ssh key.
+
+        Test for : https://pagure.io/SSSD/sssd/issue/3937
+
+        Steps:
+        1. add ipa user with ssh-key.
+        2. call sss_ssh_authorizedkeys and ssh key should be returned
+        3. check user is able to ssh or not.
+        """
+        user = 'testsshuser'
+        passwd = 'Secret123'
+        user_key = '/tmp/ssh-key'
+        # Create a user with a password
+        tasks.kinit_admin(self.master)
+        tasks.user_add(self.master, user, password=passwd)
+        user_kinit_stdin_text = "{old}\n%{new}\n%{new}\n".format(
+            old=passwd, new=passwd)
+        self.master.run_command(['kinit', user],
+                                stdin_text=user_kinit_stdin_text)
+        tasks.kinit_admin(self.master)
+        tasks.run_command_as_user(
+            self.master, user, ['ssh-keygen', '-f', user_key],
+            stdin_text='\n\n')
+        ssh_pub_key = self.master.get_file_contents('{}.pub'.format(
+            user_key), encoding='utf-8')
+        pem_file = '/tmp/cert.pem'
+        key = '/tmp/private.key'
```

Comment:
You are not using the key further in the test so there is no reason to save it. You can simply omit `-keyout`  option in openssl invocation